redis-util/src/main/java/org/sunbird/redis/RedisCacheUtil.java

```diff
@@ -0,0 +1,76 @@
+package org.sunbird.redis;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+import org.redisson.api.RMap;
+import org.redisson.api.RedissonClient;
+import org.sunbird.common.models.util.LoggerEnum;
+import org.sunbird.common.models.util.ProjectLogger;
+
+import com.typesafe.config.ConfigFactory;
+import com.typesafe.config.ConfigValue;
+
+public class RedisCacheUtil {
```

Comment:
Define a Cache interface with following methods:
* readConfig (common code available for any implementation of this interface)
* get
* put
* invalidateAll
* invalidate

RedisCache would be an implementation of this Cache interface.